archetypal/trnsys.py

```diff
@@ -425,6 +414,23 @@ def convert_idf_to_trnbuild(
     return return_path
 
 
+def add_object_and_run_ep(ep_version, idf_file, weather_file, outputs):
```

Comment:
Why did you wrap the `run_eplus` method in this new `add_object_and_run_ep()`?

Reply:
To have better coverage for trnsys.py